### PR TITLE
[Responsive] Overview

### LINKED ISSUE #1946: Responsive views: Overview
Responsive views master issue: https://github.com/decred/decrediton/issues/1820 

Preview: https://xd.adobe.com/view/f803fd3f-cbb2-412d-7b9c-b3878e8bcb5d-8e32/
Specs: https://xd.adobe.com/spec/3893e71c-63bc-41b8-77f3-6f7aa0ed6e6f-4ea2/

Starting this with the overview. All following issues will be posted in a similar format, that include these three key views. In i've not covered a specific state of a view that needs more insight, please remind me. 

Briefly the overview needs no adapation in 768px size, as the rows can simply scale down in width and the graph can be re-positioned slightly. 

In 375px mobile view however will need to replace the graph selector with a smaller variant of it, where instead of textual labels are icons. The textual graph data is positioned below the graph; and instead of 14 dates, the graph should display only 12. This way the size of it can remain the same. 

Also the navigation in the mobile view should obviously be on the bottom of the viewport and always visible above the content (just mentioning if the example is confusing).
[decrediton - graph selector icons.zip](https://github.com/decred/decrediton/files/2705383/decrediton.-.graph.selector.icons.zip)

### PR DESCRIPTION
**Pull request by vctt94**
This PR fixes #1946 

It also separates the logic from overview and tx history so it becomes easier to make it responsive. 

We still need to work on the sidebar responsiveness, but I opened an issue for that, as this PR is already getting too big.

Also we need to improve the chart responsiveness, but I am not sure if we are going to still use this package, as @alexlyp  once said he was not liking it very much, so I believe it need some discussion

### CODE REVIEW

File: app/components/views/HomePage/Page.js [hunk 1]
```diff
@@ -11,6 +11,46 @@ import TransactionsTab from "./tabs/Transactions";
...
+  <T id="home.tab.balance" m="Balance" />,
+  <T id="home.tab.tickets" m="Tickets" />,
+  <T id="home.tab.transactions" m="Transactions" />,
+];
+
+const tabLink = (i) => {
+  const m = [
+    (
+      <Tooltip text={tabMessages[i]}>
+        <span className="overview-tab-balance">&amp;nbsp;</span>
```
Comment (matheusd) [inline app/components/views/HomePage/Page.js]: Do you really need these `&amp;nbsp;` elements here and in some other spans?

File: app/style/TxHistory.less [hunk 2]
```diff
@@ -31,97 +31,109 @@
 }
 
 .pending-overview-details {
-  background-color: rgba(12, 30, 62, 0.0296);
+  background-color: #edeff1;
```
Comment (matheusd) [inline app/style/TxHistory.less]: Use a theme var for this. It's fine to define a new one if the correct one does not yet exist.

File: app/components/views/HomePage/Page.js (hunk 1 above)
Comment (vctt94) [reply app/components/views/HomePage/Page.js]: removed

File: app/style/TxHistory.less (hunk 2 above)
Comment (vctt94) [reply app/style/TxHistory.less]: fixed

**Comment by vctt94** [comment]
Yeah, it breaks the layout from the transaction history page, because they were built one above the other, which makes really hard to add responsiveness. 

On this PR I removed mostly the part which was tangled, breaking the history page. 

If you prefer I can fix on this PR, but I was thinking on fix at https://github.com/decred/decrediton/issues/1949 if that is okay to you

**Pull request merged**